java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -111,8 +112,9 @@ public static Object[] transactionTestResult() {
             1L,
             5L,
             10.5,
-            4L,
```

Comment:
why did these change?

Reply:
TransactionTestUtilities generally test commands in groups strings, sets etc... hence I added another value to Lpush to be able to remove with Lrem.